anchor/keysplit/src/lib.rs

```diff
@@ -54,7 +55,11 @@ pub fn run_keysplitter(
     // 2) Extract the validator keys from the keystore file
     info!("Extracting keys from keystore file...");
     let keys = keystore
-        .decrypt_keypair(shared.password.as_bytes())
+        .decrypt_keypair(
+            read_password(shared.password_file.as_deref())
+                .map_err(KeysplitError::Keystore)?
```

Comment:
Should we do sth like https://github.com/sigp/anchor/blob/ff56d2c17b1935105755c44cc001ee4386bdaa81/anchor/keysplit/src/lib.rs#L51 ?